types/moddable/modules/commodetto/PixelsOut.d.ts

```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright (c) 2019-2020 Bradley Farias
+*
+*   This file is part of the Moddable SDK Tools.
+*
+*   The Moddable SDK Tools is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   The Moddable SDK Tools is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with the Moddable SDK Tools.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+
+declare module "commodetto/PixelsOut" {
+    class PixelsOut {
+        constructor(width: number, height: number, pixelFormat: number);
+        begin(x: number, y: number, width: number, height: number): void;
+        end(): void;
+        continue(x: number, y: number, width: number, height: number): void;
+        send(
+            pixels: ArrayBuffer | HostBuffer,
+            offset: number,
+            count: number
+        ): void;
+
+        adaptInvalid(area: { x: number, y: number, width: number, height: number }): void;
+        pixelsToBytes(count: number): number;
+        /**
+         * The width of the PixelsOut instance in pixels.
+         */
+        readonly width: number;
+        /**
+         * 	The height of the PixelsOut instance in pixels.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/moddable/modules/commodetto/PixelsOut.d.ts;
  Line: 40;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Fixed in the latest commit.